examples/src/ref_finance.rs

```diff
@@ -320,7 +321,7 @@ async fn main() -> anyhow::Result<()> {
         actual_out
     );
     assert_eq!(actual_out, expected_return);
-    println!("Gas burnt from swapping: {}", gas_burnt);
+    println!("Gas burnt from swapping: {}", gas_burnt.as_gas());
```

Comment:
This identified the missing Display implementation: https://github.com/Mr0melian/near-gas/issues/2